app/lib/dttp/code_sets/degree_types.rb

```diff
@@ -5,6 +5,8 @@ module CodeSets
     module DegreeTypes
       NON_UK = ["Degree equivalent", "Unknown"].freeze
 
+      COMMON = ["Bachelor of Arts", "Bachelor of Science", "Master of Arts", "PhD"].freeze
```

Comment:
This is just me playing around for now! @edwardhorsford - I don't know whether we actually want to boost any of these entries to start with?

Reply:
Boosts look good 👍